app/controllers/positions_controller.rb

```diff
@@ -89,4 +91,9 @@ def position_params
                   :affiliation_type, :description, :start_date, :stop_date, :ongoing, :invite,
                   language_exp: [], project_experiences_attributes: %i[project_name _destroy id])
   end
+
+  def find_project
+    @project = Project.not_deleted.find_by('lower(name) = ? or name = ?', position_params['project_oss'].to_s.downcase,
+                                           position_params['project_oss'])
+  end
```

Comment:
Do we need to check both `lower(name)` and `name`?
I believe only `lower(name)` would suffice.